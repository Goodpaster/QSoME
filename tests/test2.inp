#Geometries taken from https://pubs.acs.org/doi/suppl/10.1021/ja4102979/suppl_file/ja4102979_si_001.pdf
subsystem
O          0.0000        0.0000        0.6565
charge -2
hl_method_num 1
end

subsystem
C          0.0000        0.0000       -0.5266
H          0.0000       -0.9326       -1.1133
H          0.0000        0.9326       -1.1133
charge +2
end

basis
 default cc-pVDZ
end

env_method_settings
 env_method b3lyp
 dynamic_level_shift
 embed_settings
  max_cycle 50
  diis_num 2
 end
end

hl_method_settings
    hl_order 1
    hl_method rhf
end

verbose 5
