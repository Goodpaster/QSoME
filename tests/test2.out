iter: 35:0               |ddm|:6.125581e-09               |Tr[DP]|:3.833595e-01
iter: 35:1               |ddm|:8.216179e-10               |Tr[DP]|:3.833595e-01

SUBSYS E
-1256.6921190846522
EMB E
11.278517892895913
PROJ E
0.38335950311487643
Subsystem 0 Energy:                                               -1245.03024169
SUBSYS E
-746.8798330231314
EMB E
-45.60280330596955
PROJ E
0.38335950300994426
Subsystem 1 Energy:                                                -496.18832392

********************************************************************************
TIMING: Freeze and Thaw                                                476.2807s
KS-DFT  Energy:                                                   -1709.61811658
TIMING: Supersystem Energy                                               0.0000s
********************************************************************************
                             Env-in-Env Calculation
********************************************************************************
Env-in-Env Energy:                                                -1708.60701456
********************************************************************************
TIMING: Env. in Env. Energy                                              0.0023s
********************************************************************************
                           HL Subsystems Calculation
********************************************************************************
Overwritten attributes  energy_elec get_fock  of <class 'pyscf.scf.uhf.UHF'>
converged SCF energy = -1248.52390349114  <S^2> = 3.7526562  2S+1 = 4.0013279
E(UCCSD) = -1249.014940436025  E_corr = -0.4910369448832432
